react-client/src/App.css

```diff
@@ -1,3 +1,16 @@
 .App {
   text-align: center;
 }
+
+a {
+  text-decoration: none !important;
+  color: #fff;
+}
+
+#title {
```

Comment:
Are we gonna stick to styling by ID? Wouldn't class be preferable?

Reply:
No, we shouldn't. Your concern is completely valid. I shall be fixing all of them